src/field/pokemon-move.ts

```diff
@@ -5,17 +5,17 @@ import { toDmgValue } from "#app/utils";
 import type { MoveId } from "#enums/move-id";
 
 /**
- * Wrapper class for the {@linkcode Move} class for Pokemon to interact with.
+ * Wrapper class for the {@linkcode MoveId} class for Pokemon to interact with.
  * These are the moves assigned to a {@linkcode Pokemon} object.
- * It links to {@linkcode Move} class via the move ID.
- * Compared to {@linkcode Move}, this class also tracks if a move has received.
+ * It links to {@linkcode MoveId} class via the move ID.
+ * Compared to {@linkcode MoveId}, this class also tracks if a move has received.
```

Comment:
```suggestion
 * Compared to {@linkcode Move}, this class also tracks if a move has received
```
I think that's what was meant in the original comment